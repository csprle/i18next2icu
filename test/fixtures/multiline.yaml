welcome: |
  Hello {{name}},
  Welcome to our platform!
  We're glad you're here.
description: |
  This is a multi-line description.
  It spans multiple lines.
  Variables like {{count}} work too.
notification_zero: No new messages
notification_one: |
  You have {{count}} new message.
  Please check your inbox.
notification_other: |
  You have {{count}} new messages.
  Please check your inbox.
error:
  title: Error occurred
  message: |
    An error has occurred: {{errorMessage}}
    Please try again later.
