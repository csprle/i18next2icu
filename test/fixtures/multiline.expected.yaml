welcome: |
  Hello {name},
  Welcome to our platform!
  We're glad you're here.
description: |
  This is a multi-line description.
  It spans multiple lines.
  Variables like {count} work too.
notification: >-
  {count, plural, =0{No new messages} one{You have {count} new message.

  Please check your inbox.} other{You have {count} new messages.

  Please check your inbox.}}
error:
  title: Error occurred
  message: |
    An error has occurred: {errorMessage}
    Please try again later.
